packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/add-node/add-node.controller.js

```diff
@@ -21,18 +22,28 @@ export default class {
   }
 
   computePrice() {
-    const flavorPrice = this.currentFlavor.nodeHourlyPrice;
+    const flavorPrice = this.showMonthlyPrices
+      ? this.currentFlavor.nodeMonthlyPrice
+      : this.currentFlavor.nodeHourlyPrice;
     let addedStorageByNode =
       this.database.disk.size - this.currentFlavor.minDiskSize;
     // if engine is distributed, the stockage is distributed between the nodes, so to get the
     // size of for one node, we need to divide by the number of nodes
     if (this.dbEngine.isDistributedStorage) {
       addedStorageByNode /= this.database.nodeNumber;
     }
-    const additionalStoragePrice = Math.max(
-      0,
-      addedStorageByNode * this.currentFlavor.hourlyPricePerGB.priceInUcents,
-    );
+    const additionalStoragePrice = this.showMonthlyPrices
+      ? Math.max(
+          0,
+          addedStorageByNode *
+            this.currentFlavor.hourlyPricePerGB.priceInUcents,
+        )
+      : Math.max(
+          0,
+          addedStorageByNode *
+            this.currentFlavor.monthlyPricePerGB.priceInUcents,
+        );
```

Comment:
Maybe this is a bit more readable:
```suggestion
  const additionalStoragePrice = Math.max(
  0, 
  this.showMonthlyPrices
    ? addedStorageByNode * this.currentFlavor.hourlyPricePerGB.priceInUcents
    : addedStorageByNode * this.currentFlavor.monthlyPricePerGB.priceInUcents
```

Reply:
Hello, yes you are right, thx! Fixed